server/src/main/java/com/linecorp/centraldogma/server/internal/api/HttpApiUtil.java

```diff
@@ -178,8 +182,11 @@ private static HttpResponse newResponse0(RequestContext ctx, HttpStatus status,
         //                 the stack trace of the cause to the trusted client.
         if (status == HttpStatus.INTERNAL_SERVER_ERROR) {
             if (cause != null) {
-                if (!(cause instanceof ShuttingDownException ||
-                      cause instanceof AbortedStreamException)) {
+                if (!(cause instanceof AbortedStreamException ||
+                      cause instanceof ClosedStreamException ||
+                      cause instanceof Http2Exception ||
+                      cause instanceof ClosedSessionException ||
```

Comment:
Can't we use `Exceptions.isExpected(Throwable)` or `Exceptions.isStreamCancelling(Throwable)` for Armeria exceptions? https://github.com/line/armeria/blob/ea2dbc05089191a4f457d03147440e3a7c27d27f/core/src/main/java/com/linecorp/armeria/common/util/Exceptions.java#L136
https://github.com/line/armeria/blob/ea2dbc05089191a4f457d03147440e3a7c27d27f/core/src/main/java/com/linecorp/armeria/common/util/Exceptions.java#L173

Reply:
That's a good idea. :-)